tpu/utils/tpu_utils.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from oauth2client.client import GoogleCredentials
+from googleapiclient import discovery
+
+credentials = GoogleCredentials.get_application_default()
+
+
+def list_tpus(project, location):
```

Comment:
Nit: Should we add documentation to function. Args/Returns?

Reply:
done